new/bahn.py

```diff
@@ -1,24 +1,27 @@
 """
 Scraper for Deutsche Bahn parking lots
 
-This scraper is disabled unless you define your API token, either in
-- environment: e.g. `export BAHN_API_TOKEN=xxx` before running the scraper
-- or in a `.env` file in the root of the scraper package containing: `BAHN_API_TOKEN=xxx`
+This scraper is disabled unless you define your DB credentials, either in
+- environment: e.g. `export DB_CLIENT_ID=xxx && export DB_API_KEY=yyy` before running the scraper
+- or in a `.env` file in the root of the scraper package containing: `DB_CLIENT_ID=xxx DB_API_KEY=yyy`
```

Comment:
Does this work? Or does it have to be two separate lines `DB_CLIENT_ID=xxx` & `DB_API_KEY=yyy`?